src/oracleFactory/updateOracle.ts

```diff
@@ -286,7 +307,10 @@ const updateOracle = ({
             { preflightCheck: false }
           )
           .catch((e) => {
-            throw new WorkflowError('Failed to match orders', e);
+            throw new WorkflowError({
+              message: 'Failed to match orders',
+              errorCause: e,
+            });
```

Comment:
are you sure there is no call to API service in the match order (ex: market API to check orders exist ? )

Reply:
good point, I removed that error, match orders does create a request order and publishes it to the market